templates/base.html.twig

```diff
@@ -1,150 +1,10 @@
-<!DOCTYPE html>
-<html style="height:100%;">
-<head>
-{% block head %}
-    <title>GRIIDC | {% block title %}Pelagos{% endblock %}</title>
+{% extends custom_base_template | orTemplateIfNotExists('griidc-base.html.twig') %}
 
-    <link rel="icon" type="image/x-icon" href="{{ asset('build/images/griidc_fav.png') }}" />
+{%block stylesheets %}
```

Comment:
These blocks are not needed, there is nothing to add or overwrite, these blocks will be inherited from the extended template. Everything works as expected.